### DISCUSSION TITLE
「自分の経験からRubyでプログラミングを教えた」を書きたい

### DISCUSSION THREAD

**Discussion by BlazingRockStorm** [rubima/magazine.rubyist.net#503] [記事を書きたい]
初めまして、

ベトナムのハノイから出身のホアン・クアンです。４年間日本に働いています。

3-4月前、私は日本語学校・専門学校の学生を少しプログラミングを教えました。そのとき、最初Python、Javascriptを教えましたが、理由があって、Rubyに変更しました。その理由を書きたいです。そして、基本のITを知らない人にプログラミングを教えるのこと（困ったこと・対策方法）をシェアしたいです。

よろしくお願いします。

**Comment by neko314** [comment]
こんにちは！
良いですね！ぜひお願いします〜🙂

記事の作成方法はご存知ですか？ 詳しくは[doc/writing_process.md](https://github.com/rubima/magazine.rubyist.net/blob/master/doc/writing_process.md)をご覧ください。わからないことがあればいつでも聞いてください👍

**Reply by BlazingRockStorm** [reply]
ありがとうございます。[doc/writing_process.md](https://github.com/rubima/magazine.rubyist.net/blob/master/doc/writing_process.md)を読んでいます。
すみませんが、質問があります。プルリクを作るために、このレポに新しいブランチを作りますか、それともこのレポをforkして、そしてforkなレポからプルリクを作りますか。

よろしくお願いします。

**Reply by neko314** [reply]
forkしてPRください👍

**Reply by BlazingRockStorm** [reply]
承知しました。ありがとうございます。
